tests/cases/fourslash/refactorConvertParamsToDestructuredObject_importedFunction3.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: f.ts
+////function foo(/*a*/a: string, b: string/*b*/) { }
+////export = foo;

```

Comment:
I think you can also `export = function(...`.  Is that case supported?

Reply:
("No" is a reasonable answer - I mostly want to know if the case was considered.)

---
In reply to: [269694751](https://github.com/Microsoft/TypeScript/pull/30475#discussion_r269694751) [](ancestors = 269694751)

Reply:
It's not supported, but I didn't considered it, that's a good point. The refactoring is not offered in this case because this is a function expression (or an arrow function) that is in an export assignment and we only offer it if the function expression is assigned to a const variable.

Reply:
i've briefly thought about supporting this case and it would require some changes to be able to call findAllReferences, so i think it's better to consider supporting this on another PR later.